bitgouel-api/src/main/kotlin/team/msg/domain/faq/mapper/FAQRequestMapper.kt

```diff
@@ -0,0 +1,8 @@
+package team.msg.domain.faq.mapper
+
+import team.msg.domain.faq.presentation.data.request.CreateFaqRequest
+import team.msg.domain.faq.presentation.web.CreateFaqWebRequest
+
+interface FaqRequestMapper {
+    fun createFAQWebRequestToDto(createFAQWebRequest: CreateFaqWebRequest): CreateFaqRequest
```

Comment:
```suggestion
    fun createFaqWebRequestToDto(createFaqWebRequest: CreateFaqWebRequest): CreateFaqRequest
```

여기 함수명과 변수명에는 소문자를 쓰는게 좋아보여요~!

Reply:
실수로 commit suggestion 안 누르고 제가 다 변경해버렸네요 ..! 수정했습니다 9081db817bfa55a8461d28fc719c863066b5a0cd d21ba80e4607291df8e61e5aa50090bfdb4d92b5